CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## v0.0.19 (unreleased)
 
+- Add `Environment.addToSetup` and `Environment.addToSandboxSetup`, making setups for `Environment`s more flexible.
```

Comment:
Seeing as we discussed making these intended for internal use should we omit this changelog entry?

Reply:
Changed the changelog entry. Is that better?